utils.py

```diff
@@ -42,7 +42,14 @@ def n(v):
         return 0L
     if v.endswith('B'):
         return long(v[:-1])
-    return long(float(v))
+    return dbus.UInt64(long(float(v)))
+
+def n32(v):
```

Comment:
I wasn't sure how ``dbus.UInt64`` would behave with a ``float`` instance instead of an integer value so I intentionally keep the casting there to make it clearer for other people reading it too.